pkg/prompt/prompt_test.go

```diff
@@ -0,0 +1,27 @@
+package prompt
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetInput(t *testing.T) {
+	tests := []struct {
+		v string
+		e bool
+	}{
+		{v: "hello\n", e: false},
+		{v: "hello", e: true},
+	}
+	for _, test := range tests {
+		input = strings.NewReader(test.v)
+		output, err := GetInput()
+		if err != nil {
+			assert.Equal(t, test.e, true)
+			continue
```

Comment:
the `continue` means you're not testing the returned data. but instead I'd change that whole `if` block to:
```
assert.Equal(t, test.e, err == io.EOF)
```
That way you are testing all cases - `err == nil` when it shouldn't be, or some non-`io.EOF` error being returned.

(and perhaps rename the `e` field to `eof`?)

Reply:
If i don't keep it within the if statement it will then run the following assert which fails `assert.Equal(t, test.v, output)` which is why i continued and ignored the 2nd assert

Reply:
Done